snooty/rstspec.toml

```diff
@@ -86,6 +86,15 @@ example = """.. warning:: ${1:Title}
    ${2:Warning block content}
 """
 
+[directive.cta-banner]
```

Comment:
Could we put this into the `mongodb` domain? i.e. `[directive."mongodb:cta-banner"]`

Reply:
Yep!